frontend/package.json

```diff
@@ -19,6 +19,7 @@
   "resolutions": {
     "nwsapi": "2.2.9",
     "d3-color": "^3.1.0",
+    "dompurify": "^3.2.4",
```

Comment:
We need to add this here to resolution to enforce that only dompurify 3.2.4 is used since there is an unused transitive dependency that required dompurify 2.x.